installer/windows/ParticleCLISetup.nsi

```diff
@@ -21,6 +21,10 @@ InstallDir "$LOCALAPPDATA\particle"
 ; File with latest CLI build
 !define ManifestURL "https://binaries.particle.io/cli/master/manifest.json"
 
+; Drivers installer
+!define DriversURL "https://binaries.particle.io/drivers/windows/particle_drivers.exe"
+!define DriversFile "particle_drivers.exe"

```

Comment:
might be nice to have an option to use the _dev_ version of the drivers as well

Reply:
In practice these change so seldom that I don't anticipate we'd use that capability often.